app/js/lib/foxbox/recipes.js

```diff
@@ -6,10 +6,35 @@
 const p = Object.freeze({
   // Private properties.
   settings: Symbol('settings'),
-  net: Symbol('net')
+  net: Symbol('net'),
+
+  // Recipe private properties.
+  service: Symbol('service')
 });
 
-export default class Recipe {
+export class Recipe {
```

Comment:
Mm just realised that I use ```[p.service]``` inside both ```Recipe``` and ```Recipes```. So I see three options:
* Leave it as is;
* Move ```Recipe``` to a separate file and expose public method like ```raw()``` that will return internal data (```this[p.service]``` now);
* Move ```Recipe``` to a separate file, leave ```[p.service]``` private, but use something like ```service: Symbol.for('recipe-raw')``` so that it can be used by both ```Recipe``` and ```Recipes```.

What do you prefer?